.github/workflows/markdown-link-check-config.json

```diff
@@ -1,13 +1,30 @@
 {
   "ignorePatterns": [
     {
-      "pattern": "^https://.*\.visualstudio\.com"
+      "pattern": "^https:\/\/.*\\.visualstudio\\.com"
```

Comment:
Backslash prior to the RegEx escaping backslash is required for JSON string encoding.